src/main/resources/db/migration/V2024_10_28_1547__CIV-15278_update_case_proceed_offline_defendant_task_item.sql

```diff
@@ -0,0 +1,20 @@
+
+/**
+ * Add scenario
+ */
+INSERT INTO dbs.scenario (name, notifications_to_delete, notifications_to_create)
+VALUES ('Scenario.AAA6.Update.CaseProceedsInCaseman.Defendant', '{}', '{"Notice.AAA6.CaseProceedsInCaseman.Defendant" : []}'),
+       ('Scenario.AAA6.Update.CaseProceedsInCaseman.Defendant.FastTrack', '{}', '{"Notice.AAA6.CaseProceedsInCaseman.Defendant" : []}');
+
+
+INSERT INTO dbs.task_item_template (task_name_en, category_en, task_name_cy, category_cy, template_name,
+                                    scenario_name, task_status_sequence, role,task_order)
+values ('<a>Upload hearing documents</a>', 'Hearing' ,'<a>Llwytho dogfennau''r gwrandawiad</a>',
```

Comment:
In the following PR: https://github.com/hmcts/civil-service/pull/5732 the task item templates _Hearing.Document.Upload, Hearing.Arrangements.Add_  would be removed. Can you check this so that they are not added again?